cvmfs/quota_posix.h

```diff
@@ -84,6 +84,11 @@ class PosixQuotaManager : public QuotaManager {
   virtual pid_t GetPid();
   virtual uint32_t GetProtocolRevision();
 
+  void ManagedReadHalfPipe(int fd, void *buf, size_t nbyte, pid_t cachemgr_pid);
+  void StoreCacheMgrPid(pid_t pid_);
+  pid_t GetCacheMgrPid();
```

Comment:
I don't think we need the getter and setter. If kept, let's inline and adjust to the style like
```suggestion
  void SetCacheMgrPid(pid_t pid_) { ... }
  pid_t cache_mgr_pid() const { return ... }
```